AzureTable/Microsoft.DataTransfer.AzureTable.FunctionalTests/packages.config

```diff
@@ -1,16 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Castle.Core" version="3.3.3" targetFramework="net451" />
-  <package id="Microsoft.Azure.CosmosDB.Table" version="0.9.0-preview" targetFramework="net451" />
+  <package id="Microsoft.Azure.CosmosDB.Table" version="2.0.0" targetFramework="net451" />
   <package id="Microsoft.Azure.DocumentDB" version="1.19.0" targetFramework="net451" />
```

Comment:
I thought this is the actual CosmosDB SDK. Does it need a version bump as well? There is an open issue where it fails to connect to local emulator and I was wondering, if it will be resolved when we update the SDK?

Reply:
This is not the cosmosDB SDK. That's the : Microsoft.Azure.DocumentDB
This is only updating the CosmosDB Tables SDK (different from the cosmosDB SDK).